examples/OpenCL-cpp/CMakeLists.txt

```diff
@@ -7,7 +7,10 @@ include("../common.cmake")
 project(download_opencl_cpp)
 
 hunter_add_package(OpenCL-cpp)
-find_package(OpenCL-cpp CONFIG REQUIRED)
+find_package(OpenCLHeadersCpp CONFIG REQUIRED)
+
+hunter_add_package(OpenCL)
```

Comment:
could you explain why we need to manually add OpenCL-cpp and OpenCL? Maybe leave the explanation as a comment in the example and additionally maybe add that to the documentation

Reply:
OpenCL is just the C headers, these are the CPP headers (which depend on the C headers). As usual with KhronosGroup libraries, the naming and interdependencies are confusing. 😕